sci-visualization/circos/circos-0.69.9.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV=$(ver_rs 2 '-')
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Circular layout visualization of genomic and other data"
+HOMEPAGE="http://mkweb.bcgsc.ca/circos/"
+SRC_URI="http://mkweb.bcgsc.ca/circos/distribution/${PN}-${MY_PV}.tgz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S="${WORKDIR}/${MY_P}"
+
+DEPEND="dev-perl/Config-General
+	dev-perl/GD
+	dev-perl/Math-Bezier
+	dev-perl/Math-Round
+	dev-perl/Math-VecStat
+	dev-perl/Params-Validate
+	dev-perl/Readonly
+	dev-perl/Regexp-Common
+	>=dev-perl/Set-IntSpan-1.11
+	dev-perl/Graphics-ColorObject
+	dev-perl/List-MoreUtils
+	dev-perl/Clone
```

Comment:
In proxy-maint we usually sort the list alphabetically, but this is not proxy-maint'd package so just follow the previous style. Just letting you know in case you added this list, or updated it without sorting.